etsd/keys/forms.py

```diff
@@ -43,6 +43,10 @@ def clean(self):
 
         return data
 
+class PublicKeyAcceptRejectForm(forms.ModelForm):
+    class Meta:
+        model = models.PublicKey
+        fields = ("status",)
```

Comment:
To make this field hidden you can add a HiddenInput widget:  https://docs.djangoproject.com/en/3.2/ref/forms/widgets/#hiddeninput